DirectProgramming/C++SYCL_FPGA/Tutorials/Features/multi_aocx/device_code_split/src/simple.cpp

```diff
@@ -0,0 +1,85 @@
+#include <sycl/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <iostream>
+
+using namespace sycl;
+constexpr access::mode sycl_read = access::mode::read;
+constexpr access::mode sycl_write = access::mode::write;
+
+class SimpleAdd;
+class SimpleMul;
+
+void kernel_vadd(const float a, const float b, float &c) {
+
+  {
+#if FPGA_SIMULATOR
+    auto selector = sycl::ext::intel::fpga_simulator_selector_v;
+#elif FPGA_HARDWARE
+    auto selector = sycl::ext::intel::fpga_selector_v;
+#else // #if FPGA_EMULATOR
+    auto selector = sycl::ext::intel::fpga_emulator_selector_v;
+#endif
+
+    queue deviceQueue(selector);
```

Comment:
We usually want code samples to use simpler names for things like queues and handlers and to follow the [google coding style guide](https://google.github.io/styleguide/cppguide.html) - so review things like case of variables, function names etc.
The external review will also request these changes so might as well make it now.

```suggestion
    queue q(selector);
```
and replace all uses below